run-cf-tests.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+#set -x
+set -o pipefail
+
+### This script runs the Checker Framework's tests that depend on
+### dljc (this project). It is intended to be used as part of a CI integration test.
+### This script must be run from the top-level directory
+### It performs these actions:
+### 1. fetches plume-scripts
+### 2. uses git-clone-related to clone either the Checker Framework master version or the current user's
+###    fork of the CF on the same branch, if it exists
+### 3. re-clones this repo/branch in the clone of the CF at its checker/bin/.do-like-javac directory
+### 4. runs the CF tests that use dljc
+
+### get plume-scripts
+
+if [ -d /tmp/"$USER"/plume-scripts ] ; then
+  git -C /tmp/"$USER"/plume-scripts pull -q > /dev/null 2>&1
+else
+  mkdir -p /tmp/"$USER" && git -C /tmp/"$USER" clone --depth 1 -q https://github.com/plume-lib/plume-scripts.git
+fi
+
+### clone the CF
+
+/tmp/"$USER"/plume-scripts/git-clone-related typetools checker-framework /tmp/"$USER"/checker-framework
```

Comment:
This will use whatever is currently at `/tmp/"$USER"/checker-framework` if it exists.  I think you want to `rm -rf` that directory before running this command, so that the results of this script are reproducible.